<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="dell-PC" ignored="0" name="FaiureTest" tests="1" failures="1" timestamp="2020-05-14T07:34:01 IST" time="0.173" errors="0">
  <testcase name="doLogin" time="0.021" classname="Customlister.TestFailure">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Customlister.TestFailure.doLogin(TestFailure.java:15)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
</testsuite> <!-- FaiureTest -->
